content/en/docs/contribute/localization.md

```diff
@@ -361,6 +361,20 @@ extensive human review to meet minimum standards of quality.
 To ensure accuracy in grammar and meaning, members of your localization team
 should carefully review all machine-generated translations before publishing.
 
+### Translating SVG Images
+
```

Comment:
```suggestion

The Kubernetes project recommends using vector (SVG) images where possible, as these
are much easier for a localization team to edit. If you find a raster image that needs localizing,
consider first redrawing the English version as a vector image, and then localize that.

```

Reply:
@mboukhalfa what did you want to do with this feedback? You haven't said.

Reply:
You could add this paragraph, if you wish to:
```markdown
The Kubernetes project recommends using vector (SVG) images where possible, as these
are much easier for a localization team to edit. If you find a raster image that needs localizing,
consider first redrawing the English version as a vector image, and then localize that.
```

Reply:
Thank you for pointing out the missed paragraph. I appreciate your suggestion, and I added it to the PR.